recOrder/plugin/workers/calibration_workers.py

```diff
@@ -1,3 +1,6 @@
+# TODO: remove in Python 3.11
+from __future__ import annotations
+
```

Comment:
Needed?

Reply:
Doesn't change behavior. But some type hints will have to be strings without this, which is aesthetically suboptimal.